src/taglib/taglib-loader/Tag.js

```diff
@@ -269,6 +280,10 @@ class Tag {
         return (this._nodeFactory = nodeFactory);
     }
 
+    isCustomTag() {
+        return this.renderer || this.template || this.isNestedTag;
```

Comment:
Not sure how I feel about this function, `nestedTags` are not custom tags :p.

Not sure if we'll still need this if you switch to checking for the specific taglib id defining the `key` attribute though.